homework/index.js

```diff
@@ -30,15 +15,162 @@
     return elem;
   }
 
+  function remove() {
+    document.getElementById('right_column_list').remove();
+  }
+
+  function setLink(linkID, srcArray) {
+    document.getElementById(linkID).innerText = srcArray.name;
+    document.getElementById(linkID).setAttribute('href', srcArray.html_url);
+    document.getElementById(linkID).setAttribute('target', '_blank');
+  }
+
+  const root = document.getElementById('root');
```

Comment:
I would suggest you move all these statements to its own function which you can call from the `main()`. And if you are using some of these elements in other places, you can then try return those elements or do a `document.getElementById` in areas where you need them.

Reply:
Actually, I was trying to empty main function in order to get more pure code. But, in the end there is no need for this. Included them again.